experimental/trusted_shuffler/scripts/run_trusted_shuffler

```diff
@@ -0,0 +1,8 @@
+#!/usr/bin/env bash
+
+RUST_LOG=info cargo run --manifest-path=experimental/trusted_shuffler/server/Cargo.toml -- \
+--k=1 --listen-address="[::]:8080" --backend-url="http://localhost:8888" &
+   
+# Convenient to know the PID to kill when the Trusted Shuffler crashes.
+echo "Trusted Shuffler PID: "
+echo $!
```

Comment:
(optional) Not sure I see the point of having a script that spawns the command in the background. Why not just make this script `exec` the command, and the caller of the script can decide to run it in the background if they want to?

Reply:
Done, thanks, it was an oversight!